vms/avm/unique_tx.go

```diff
@@ -57,11 +45,26 @@ type TxState struct {
 	status choices.Status
 }
 
+func newUniqueTx(vm *VM, txID ids.ID, rawTx *Tx) *UniqueTx {
```

Comment:
`// newUniqueTx returns the UniqueTx representation of transaction [txID]. rawTx may be nil.`